crates/iota-rest-api/openapi/openapi.json

```diff
@@ -4054,11 +4125,16 @@
             "description": "Whether the system is running in a downgraded safe mode due to a non-recoverable bug. This is set whenever we failed to execute advance_epoch, and ended up executing advance_epoch_safe_mode. It can be reset once we are able to successfully execute advance_epoch.",
             "type": "boolean"
           },
-          "safe_mode_computation_rewards": {
+          "safe_mode_computation_charges": {
             "description": "Amount of computation rewards accumulated (and not yet distributed) during safe mode.",
             "type": "string",
             "format": "u64"
           },
+          "safe_mode_computation_charges_burned": {
+            "description": "Amount of burned computation rewards accumulated during safe mode.",
```

Comment:
```suggestion
            "description": "Amount of burned computation charges accumulated during safe mode.",
```

Reply:
This suggestion will be outdated due to https://github.com/iotaledger/iota/pull/5270/commits/3d65d4a39724a007ab66be516b08e4bb0fc44c9d and https://github.com/iotaledger/iota/pull/5270/commits/3f5933339b60e7b381332c09e578516965fc801c

`IotaSystemStateSummaryV1` will now be kept unchanged with field `safe_mode_computation_rewards` and `IotaSystemStateSummaryV2` introduced with renamed and additional fields.

Reply:
fixed with https://github.com/iotaledger/iota/pull/5276/commits/1ef7937537026cbeba44fa011f55458cdb1bc851